src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/CommandOutputLogger.cs

```diff
@@ -12,20 +12,14 @@ namespace NuGet.CommandLine.XPlat
     /// <summary>
     /// Logger to print formatted command output.
     /// </summary>
-    internal class CommandOutputLogger : LegacyLoggerAdapter, ILogger
+    internal class CommandOutputLogger : LoggerBase, ILogger
```

Comment:
I get that the change was made here, but I'm not sure what the requirement behind this change is.

What would we lose if were to keep this implementation?

Reply:
The reason for this change is to add [error code in the error messages](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/Logging/LoggingExtensions.cs#L16) to match `nuget.exe` messages that are printed to console.

Reply:
> What would we lose if were to keep this implementation?

The error messages printed to console will not have any error codes. I thought it would be easy for users to search for `error code` in docs.

Reply:
I'm curious why `LegacyLoggerAdapter` was implemented instead of `LoggerBase`, but if no tests are failing, I'm hoping it's all good.